lib/irule-tester

```diff
@@ -10,7 +10,7 @@
 
 # irule-tester version number
 
-typeset -r version='0.6.0'
+typeset -r version='0.6.1'
```

Comment:
Since I'm using SEMVER standards on this project, and this pull request adds new features, let's bump the minor version instead of the bugfix version.